src/www/ui/search-helper.php

```diff
@@ -36,7 +36,7 @@
  *         contains uploadtree_pk, parent, upload_fk, pfile_fk, ufile_mode, and
  *         ufile_name
  */
-function GetResults($Item, $Filename, $tag, $Page, $SizeMin, $SizeMax, $searchtype,
+function GetResults($Item, $Filename, $Upload, $tag, $Page, $SizeMin, $SizeMax, $searchtype,
```

Comment:
I just realized that this function is used by REST API as well and this will cause failure for it.

Can you please add additional header for the `uploadId` in `SearchController`?

Reply:
Good point, I completely forgot about RAST API.
Of course I'll add it.

The thing is that when there is no uploadId supplied, SearchController should send `0` as `$Upload` parameter.
Unless you have another method, I will do it this way.

Reply:
Done

Reply:
Awesome!